src/lib.rs

```diff
@@ -98,6 +98,10 @@ pub struct Message {
     /// The message itself.
     #[serde(rename = "m")]
     pub message: String,
+
+    /// additional params
+    #[serde(rename = "p")]
+    pub params: HashMap<String, String>,
```

Comment:
It probably makes sense to call this "extra_tags" since we have based this crate on the [Django implementation](https://github.com/django/django/blob/c4df2a77761a1ae392eb5c4803b5712803d5239f/django/contrib/messages/storage/base.py#L18).